android/src/main/java/com/reactnativecommunity/cookies/CookieManagerModule.java

```diff
@@ -102,6 +102,20 @@ public void flush(Promise promise) {
         }
     }
 
+    @ReactMethod
+    public void removeSessionCookies(Promise promise) {
+        try {
+            getCookieManager().removeSessionCookies(new ValueCallback<Boolean>() {
+                @Override
+                public void onReceiveValue(Boolean data) {
+                    promise.resolve(data);
+                }
+            });
+        } catch (Exception e) {
+            promise.reject(e);
+        }
+    }
```

Comment:
Do we need to flush here?

Reply:
Not flush, sync*

Reply:
From what I've seen so far, we didn't have to do a flush or sync or anything else to get fetch or a webview to get rid of these session cookies. We've just been using this call and the next fetch we do results in the session cookies no longer being there.

Reply:
sg! lets ship it